js-intro.js

```diff
@@ -43,15 +59,25 @@ var myCat = "Garfield"
 
 // ------------------------------ Consider the following variable:
 
-var myMessage = "Learning is fun!"
+// var myMessage = "Learning is fun!"
 
 // 3a. Write the code that logs each letter of the message using a for loop.
 //hint: "logs" in this case means "prints to the terminal console."  This is typically done with a console.log(x), where x is the string, function, etc that you wish to see in the terminal.
 
-
+var myMessage = "Learning is fun!"
+const myMessage2 = (string) =>{
+let newVar = myMessage
+for(let i=0; i< myMessage.length; i++){
+}
+return newVar
+}
+    console.log(myMessage2(myMessage))
+    
```

Comment:
This just returns the string as is. To return every letter you would want to do something like:
const myMessage2 = (string) =>{
  for(let i=0; i< string.length; i++){
    console.log(string[i])
  }
}
myMessage2(myMessage)